x/fundraising/client/cli/flags.go

```diff
@@ -28,7 +28,7 @@ func flagSetBids() *flag.FlagSet {
 	fs := flag.NewFlagSet("", flag.ContinueOnError)
 
 	fs.String(FlagBidderAddr, "", "The bech32 address of the bidder account")
-	fs.String(FlagIsWinner, "", "The eligibility of the bidder to purchase the selling coin")
+	fs.String(FlagIsMatched, "", "The eligibility of the bidder to purchase the selling coin")
```

Comment:
This description may mislead bidders. Should we update the comment? 

Something like the following:

> The bid that successfully matched (a.k.a., winner)

Reply:
I changed as suggested.